core/templates.yml

```diff
@@ -158,6 +158,20 @@ keys:
             ccc: ccc
         default: cc
         alias: extension
+    zbrush.ext:
+        type: str
+        choices:
+            ztl: ztl
+            ZTL: ZTL
+        default: ztl
+        alias: extension
+    substance_painter.ext:
```

Comment:
We can leave this just as `substance.ext` instead of `substance_painter.ext` if we need to support other substance file formats we can add those later.

Reply:
ok
i will update